docs/templates/getting-started/faq.md

```diff
@@ -598,3 +603,19 @@ K.set_session(sess)
 
 # Rest of code follows ...
 ```
+
+### How can I install HDF5 or h5py to save my models in Keras?
+
+In order to save your Keras models as hdf5 files, e.g. via `keras.callbacks.ModelCheckpoint`, Keras uses the h5py Python package. They are a dependency of Keras and should be installed by default. On Debian based distributions you will have to additionally install
```

Comment:
- Use ~80 char lines to make the changes more amenable to version control (which works line-by-line)
- hdf5 -> HDF5
- "They are" -> "It is"
- "On Debian-based distributions, you will have to additionally install `libhdf5`:"